erpnext/hr/doctype/employee_grievance/employee_grievance.json

```
@@ -0,0 +1,292 @@
+{
+ "actions": [],
+ "autoname": "HR-GRIEV-.YYYY.-.#####",
+ "creation": "2021-05-11 13:41:51.485295",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "raised_by",
+  "designation",
+  "reports_to",
+  "column_break_3",
+  "date",
+  "status",
+  "grievance_details_section",
+  "grievance_against_party",
+  "grievance_against",
+  "grievance_type",
+  "column_break_11",
+  "associated_document_type",
+  "associated_document",
+  "section_break_14",
+  "description",
+  "cause_of_grievance_section",
+  "cause_of_grievance",
+  "section_break_15",
+  "is_applicable_for_pay_cut",
+  "is_applicable_for_suspension",
+  "number_of_days_for_suspension",
+  "column_break_20",
+  "suspended_from",
+  "suspended_to",
+  "unsuspended_on",
+  "resolution_details_section",
+  "resolved_by",
+  "resolution_date",
+  "employee_responsible",
+  "column_break_16",
+  "resolution_detail",
+  "amended_from"
+ ],
+ "fields": [
+  {
+   "fieldname": "grievance_type",
+   "fieldtype": "Link",
+   "in_list_view": 1,
+   "label": "Grievance Type",
+   "options": "Grievance Type",
+   "reqd": 1
+  },
+  {
+   "fieldname": "column_break_3",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "date",
+   "fieldtype": "Date",
+   "in_list_view": 1,
+   "label": "Date ",
+   "reqd": 1
+  },
+  {
+   "default": "Open",
+   "fieldname": "status",
+   "fieldtype": "Select",
+   "in_list_view": 1,
+   "label": "Status",
+   "options": "Open\nInvestigated\nResolved\nInvalid",
```

```suggestion
   "options": "Open\nInvestigation\nResolved\nInvalid",
```
IMO, "Investigated" denotes a past action, so we should have the status as "Investigation" since this will track that grievance is under investigation.

---

https://github.com/frappe/erpnext/pull/25705#discussion_r635061636
As mentioned in this comment I think we should keep it Investigated as we need to track the root cause too...